.travis.yml

```diff
@@ -4,8 +4,8 @@ sudo: false
 dist: trusty
 
 rvm:
-  - 2.3.4
-  - 2.4.2
+  - 2.5.0
+  - 2.6.0
```

Comment:
This isn't critical, but do you mind changing to 2.6.4 because that's our preferred 2.6.x ruby version on the deployed servers, and it's kind of nice to test on a later available ruby version that may have patched bugs in earlier patch versions?